mobify/progressive-web-scaffold
# Astro: merlins add segmented view (tabs) into the account tab

Creates native tab on account tab that switches between registration and login page.
This is supplement to #274

 **JIRA**: https://mobify.atlassian.net/browse/HYB-982
 **Linked PRs**: https://github.com/mobify/progressive-web-scaffold/pull/274, #278 

## Changes
- on account tab instead of normal navigationView, it leads to a new view with tabs and webview. 

## How to test-drive this PR
- start the app, go on the account tab. Switch between the tabs on the top. It should load the correct page
